integration_test/pg/explain_test.exs

```diff
@@ -26,4 +26,19 @@ defmodule Ecto.Integration.ExplainTest do
       TestRepo.explain(:all, Post, analyze: "1")
     end)
   end
+
+  test "explain JSON format" do
+    [explain] = TestRepo.explain(:all, Post, analyze: true, verbose: true, timeout: 20000, format: :json) |> Jason.decode!
```

Comment:
Considering this comment https://github.com/elixir-ecto/ecto_sql/pull/295#discussion_r554882472, I'd suggest `Jason.decode!()` :)